### ISSUE TITLE
Resolving stub yaml path fails if multiple yaml paths are provided

### ISSUE THREAD

**Issue by dukeofharen** [dukeofharen/httplaceholder#50]
Let's say you have the following stub

```yaml
- id: stub1
  conditions:
    method: GET
    url:
      path: /img1
  response:
    file: 1.jpg
    headers:
      Content-Type: image/jpeg
```

This stub is located in folder `/var/stubs/1`. There is also a folder `/var/stubs/2` that you want to read with HttPlaceholder.

When you start the application like this: `httplaceholder --inputFile /var/stubs/1%%/var/stubs/2` and you go to `http://httplaceholder-host:5000/img1`, the image will not be resolved because the provided `inputFile` parameter is not a valid path. This should be the case for HttPlaceholder to find the file `1.jpg` (which is located besides the stub file).

In short, what needs to happen, every path specified in `inputFile` should be checked iterated to find the specified file.
